RELEASE_NOTES.md

```
@@ -4,125 +4,317 @@
 
 ---
 
-# [2.2.0]
+# [2.3.0] - Recommended Version Bump: Minor (2.2.0 → 2.3.0)
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)